pycytominer/cyto_utils/features.py

```diff
@@ -175,7 +175,7 @@ def drop_outlier_features(
 
     # Subset dataframe
     if samples != "all":
-        population_df.query(samples, inplace=True)
+        population_df = population_df.query(samples)
```

Comment:
Consider adding a comment here or replacing the one above this if-branch to better describe what's happening here and why. For example, the comment in `correlation_threshold` provides more information (but may also need to be updated).